rusty_pizza_server/src/order_model/meal.rs

```diff
@@ -59,6 +59,10 @@ impl Meal {
     pub fn remove_special(&mut self, special: &String) {
         self.specials.remove(special);
     }
+
+    pub fn get_price(&self) -> Money {
+        self.price
```

Comment:
`Money` is immutable, but I think we should make a defensive copy here:
```suggestion
        self.price.clone()
```
as `Money` represents a "value type" I would prefer returning a copy over returning a reference.